docs_src/security/Ch-APIGateway.md

```diff
@@ -2,55 +2,79 @@
-
-KONG (<https://konghq.com/>) is the product underlying the API gateway.
-The EdgeX community has added code to initialize the KONG environment,
-set up service routes for EdgeX microservices, and add various
-authentication/authorization mechanisms including JWT authentication and ACL.
+!!! edgey "EdgeX 3.0"
+    This content is completely new for EdgeX 3.0.
+    EdgeX 3.0 uses a brand new API gateway solution
+    based on NGINX and Hashicorp Vault instead of Kong and Postgres.
+    The new solution means that EdgeX 3.0 will be
+    able to run in security enabled mode on more resource-constrained devices.
+
+API gateways are used in microservice architectures
+that expose HTTP-accessible APIs to create a
+security layer that separates internal and external callers.
+An API gateway accepts client requests,
+authenticates the client,
+forwards the request to a backend microservice,
+and relays the results back to the client.
+
+Although authentication is done at the microservice layer in EdgeX 3.0,
+EdgeX Foundry as elected to continue to use an API gateway for the
+following reasons:
+
+1. It provides a convenient choke point and policy enforcement point
+   for external HTTP requests and enables EdgeX adopters to
+   easily replace the default authentication logic.
+
+2. It defers the urgency of implementing fine-grained authorization at
+   the microservice layer.
+
+3. It provides defense-in-depth against microservice authentication bugs
+   and other technical debt that might otherwise put EdgeX
+   microservices at risk.
+
+The API gateway listens on two ports:
+
+* 8000: This is an unencrypted HTTP port exposed to localhost-only
+  (also exposed to the edgex-network Docker network).
+  When EdgeX is running in security-enabled mode,
+  the EdgeX UI uses port 8000 for authenticated
+  local microservice calls.
+
+* 8443: This is a TLS 1.3 encrypted HTTP port exposed via
+  the host's network interface to external clients.
+  The default TLS certificate on this port is untrusted
+  by default and should be replaced with a trusted
+  certificate for production usage.
+
+EdgeX 3.0 uses NGINX as the API gateway implementation
+and delegates to EdgeX's secret store (powered by Hashicorp Vault)
+for user and JWT authentication.
+
 
 ## Start the API Gateway
 
-The API gateway is started by default when using 
-the secure version of the Docker Compose files found at
-<https://github.com/edgexfoundry/edgex-compose/tree/ireland>.
+The API gateway is started by default in either
+the snap-based EdgeX deployment
+or the Docker-based EdgeX deployment
+using the Docker Compose files found at
+<https://github.com/edgexfoundry/edgex-compose/>.
 
-The command to start EdgeX inclusive of API gateway related services is:
+In Docker, the command to start EdgeX inclusive of API gateway related services is:
 
-    git clone -b ireland https://github.com/edgexfoundry/edgex-compose
+    git clone -b jakarta https://github.com/edgexfoundry/edgex-compose
```

Comment:
jakarta?

Reply:
Changed to "somerelease" (with note explaining it)